### ISSUE TITLE
Save and Reuse of DDPG model

### ISSUE THREAD

**Issue by lyjge** [MorvanZhou/Reinforcement-learning-with-tensorflow#131]
Hello Dr. MorvanZhou,

Thanks for sharing this great repository.

May I ask you 2 simple (stupid, as I am a rookie of RL~lol) questions:

1. How can I save the trained model of your `DDPG_update2.py` code.

2. Then how can I load and reuse the saved model to get action and the new state (when doing this the model is not going to be trained).

Many Thanks,
YL

**Comment by jandevries123** [comment]
Hi lyjge,

I acctually got that setup for the DDPG Code, both. If you could figure how to add batch normalization in the Actor network between L1 and L2 i'd be happy to share, so we could help each other?

Jan
